0.X/Dockerfile

```diff
@@ -6,7 +6,8 @@ ARG VAULT_VERSION=1.4.1
 # Create a vault user and group first so the IDs get set the same way,
 # even as the rest of this may change over time.
 RUN addgroup vault && \
-    adduser -S -G vault vault
+    adduser -S -G vault vault && \
+    addgroup vault root
```

Comment:
I'm curious why the primary group for `vault` is set to `vault` here, but then $HOME and /vault are being chgrp'ed to 0 below. i.e. should this be setting the primary group to 0?

Reply:
This is really for backwards compatibility.  Vault users was already part of the `vault` group since this Docker image launched, so we don't want to change too much about ownership of files on persistent volumes that already exist.